apps/demo/src/app/cart/components/add-to-cart-notification/components/product-added/product-added.component.html

```diff
@@ -2,7 +2,7 @@
   <img *ngIf="product.images" src="{{product.images[0].url}}">
   <div class="demo-product-added__description">
     <div class="demo-product-added__name">{{product.name}}</div>
-    <div class="demo-product-added__price">${{product.price}}</div>
+    <div class="demo-product-added__price">${{product.price.originalPrice}}</div>
```

Comment:
I believe this SHOULD be discounted price, right?

Reply:
changed